CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ list(APPEND CMAKE_PREFIX_PATH /usr/local)
 #  -fwrapv              -> Make signed integer overflows defined
 #  -fvisibility=hidden	-> Force defaultsymbol visibility to hidden
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall -fno-strict-aliasing -fwrapv -fvisibility=hidden")
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Why do we need this?

Reply:
otherwise I get a relocation linkage error.